chains/evm/calls/bridge/deposit.go

```diff
@@ -0,0 +1,30 @@
+package bridge
+
+import (
+	"github.com/ethereum/go-ethereum/common/math"
+	"math/big"
+)
+
+func ConstructErc20DepositData(destRecipient []byte, amount *big.Int) []byte {
```

Comment:
I moved these calls to a separate file, as there is no reason to keep them coupled with `BridgeContract`, and also `bridge.go` file is already pretty big.

Reply:
yeap, but i would also move them to something like depostis package